index.html

```diff
@@ -16,6 +16,11 @@
             </p>
             <nav class="nav">
                 <i class="fa-solid fa-bars"></i>
+                <ul class="menu">
+                    <li><a href="http//">Portofolio</a></li>
+                    <li><a href="http//">About me</a></li>
+                    <li><a href="http//">Contact</a></li>
+                </ul>
```

Comment:
- [ ] Great work implementing the navbar llinks 👍. I think there is a slight discrepancy when it comes to the design though in the `font-family` and the `font-weight`. I've added an image below for comparison, kindly change it to mimic the Figma design
> Your Design
![image](https://user-images.githubusercontent.com/90556221/177742180-a2c8970c-1602-4f80-be06-a25c30d730aa.png)

> Figma Design
![image](https://user-images.githubusercontent.com/90556221/177742266-b16d0f76-9ce7-4e17-be83-6a2f521c2d6b.png)